src/base/perf_monitor.c

```diff
@@ -4489,3 +4509,70 @@ perfmon_get_peek_stats (UINT64 * stats)
   stats[pstat_Metadata[PSTAT_HF_NUM_STATS_ENTRIES].start_offset] = heap_get_best_space_num_stats_entries ();
   stats[pstat_Metadata[PSTAT_QM_NUM_HOLDABLE_CURSORS].start_offset] = session_get_number_of_holdable_cursors ();
 }
+
+/*
+ * print_timer_to_file - Print in a file the statistic values for a timer type statistic
+ *
+ * stream (in/out): input file
+ * stat_index (in): statistic index
+ * stats_ptr (in): statistic values array
+ *
+ * return: void
+ *
+ */
+static void
+print_timer_to_file (FILE * stream, int stat_index, UINT64 * stats_ptr)
```

Comment:
I think it's better to format the print instead of:
> bt_traverse                   = 1282992351

all timer values:
> Num_bt_traverse          = xxxxxxxx
   Total_time_bt_traverse  = xxxxxxxx
   Max_time_bt_traverse   = xxxxxxxx
   Avg_time_bt_traverse    = xxxxxxxx

Print as:
```c
snprintf (p, remained_size, "Num_%-25s = %10llu\n", pstat_Metadata[i].stat_name, ...);
...
snprintf (p, remained_size, "Total_time_%-18s = %10llu\n", pstat_Metadata[i].stat_name, ...);
...
```
Consider the prefix when setting the printed string length.